baseframe/templates/baseframe/bootstrap3/ajaxform.html.jinja2

```diff
@@ -1,6 +1,8 @@
 {% from "baseframe/bootstrap3/forms.html.jinja2" import renderform, ajaxform, widget_ext_scripts, widgetscripts %}
 {{ renderform(form=form, formid=formid, ref_id=ref_id, submit=submit, message=message, cancel_url=cancel_url, multipart=multipart) }}
-{{ ajaxform(ref_id=ref_id, request=request, force=true) }}
+{%- if with_chrome -%}
+  {{ ajaxform(ref_id=ref_id, request=request, force=true) }}
+{%- endif -%}
```

Comment:
This appears to force ajaxform on all forms even if the app did not want it. This may break non-Ajax forms in some apps. Confirm, please?